Week_2/functions/updateIfGreater/updateifgreater.cpp

```diff
@@ -0,0 +1,24 @@
+#include <iostream>
+
+using namespace std;
+
+void UpdateIfGreater(const int& source, int& destination)
```

Comment:
There was a hint about references for types. Generally you're right about const& on source but for base types like int passing by value is faster than using const&.

Reply:
where can i read about it?

Reply:
Technical interviews on c++ position. =) To be honest I don't know. Probably ravesli.com in references section.